src/core/ui.js

```diff
@@ -101,17 +101,17 @@ function trapFocus(element) {
   const lastFocusableEl = focusableEls[focusableEls.length - 1];
 
   element.addEventListener("keydown", e => {
-    if (e.key !== "Tab" || e.keyCode !== /*tab keycode*/ 9) {
+    if (e.key !== "Tab" || e.keyCode !== /* tab keycode */ 9) {
```

Comment:
Why keyCode? We don't support oooold IE, right?

Reply:
No, we don't... if this is just for IE, we can ditch.